### PR TITLE
Correct package name in doc comment

### PR DESCRIPTION
**Pull request by tamird**
@ianlancetaylor

### CODE REVIEW

File: backtrace.h [hunk 1]
```diff
@@ -7,13 +7,13 @@ modification, are permitted provided that the following conditions are
 met:
 
     (1) Redistributions of source code must retain the above copyright
-    notice, this list of conditions and the following disclaimer.
```
Comment (ianlancetaylor) [inline backtrace.h]: Thanks, but the files like backtrace.h and so forth are simply copied from the libbacktrace library in GCC.  I don't particularly want to change them, except perhaps to update them with a new copy from GCC.  If that is done it should be in a change by itself with no other changes.

Comment (tamird) [reply backtrace.h]: OK; removed this bit and will send a separate PR with the updates from upstream.

**Comment by tamird** [comment]
@ianlancetaylor any reason not to merge this?

**Comment by ianlancetaylor** [comment]
I would like the commit message to be

    cgosymbolizer: correct package name in doc comment

Thanks.

**Comment by tamird** [comment]
Done.

**Comment by ianlancetaylor** [comment]
Thanks.

**Pull request merged**